### ISSUE TITLE
Program runtime issues

### ISSUE THREAD

**Issue by nomoneyExpection** [IDEA-Research/Grounded-Segment-Anything#331]
I would like to ask everyone who is using this project how long it takes on average to run **Grounded-SAM: Detect and Segment Everything with Text Prompt.** It takes me about two and a half minutes. Is it because of my computer configuration problem or other problems or Where is the time spent running the program?

**Comment by rentainhe** [comment]
You can refer to this issue https://github.com/IDEA-Research/Grounded-Segment-Anything/issues/307 to see if there're some accelerate method for solving this

**Comment by nomoneyExpection** [comment]
No, it takes me two and a half minutes to run the first grounded_sam_demo.py, and the results are as follows：

(grounded_segment_anything) E:\forwork\Grounded-Segment-Anything>python grounded_sam_demo.py --config GroundingDINO/groundingdino/config/GroundingDINO_SwinT_OGC.py --grounded_checkpoint groundingdino_swint_ogc.pth --sam_checkpoint sam_vit_h_4b8939.pth --input_image assets/demo1.jpg --output_dir "outputs" --box_threshold 0.3 --text_threshold 0.25 --text_prompt "bear" --device "cuda" E:\anaconda\envs\grounded_segment_anything\lib\site-packages\torch\functional.py:568: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  ..\aten\src\ATen\native\TensorShape.cpp:2228.)   return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined] final text_encoder_type: bert-base-uncased Some weights of the model checkpoint at bert-base-uncased were not used when initializing BertModel: ['cls.seq_relationship.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.bias', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.dense.weight', 'cls.seq_relationship.bias'] - This IS expected if you are initializing BertModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model). - This IS NOT expected if you are initializing BertModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model). _IncompatibleKeys(missing_keys=[], unexpected_keys=['label_enc.weight']) E:\anaconda\envs\grounded_segment_anything\lib\site-packages\transformers\modeling_utils.py:884: FutureWarning: The `device` argument is deprecated and will be removed in v5 of Transformers.   warnings.warn( E:\anaconda\envs\grounded_segment_anything\lib\site-packages\torch\utils\checkpoint.py:25: UserWarning: None of the inputs have requires_grad=True. Gradients will be None   warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
